docs/Barista-WorlFlow.md

```diff
@@ -0,0 +1,45 @@
+# Barista Project Workflow 
+In the follwing pages, you will find all the information necessary to navigate and effectively start working on any of the Barista projects. 
+Every single one of them  must go through the following stages: First, the project is defined and scope, followed by the planning, execution  and finally, the review. For each stage, there are some specific tools and protocols that must be followed:
+ 
+## Table of contents 
+1. [Definining and scoping](#scope)
+    1. [Basic Info](#basic)
+2. [Planning and estimating](#planning)
+    1. [Pivotal](#pivotal)
+     2. [Repository structure](#structure)
+    3. [Pull Requests](#pr)
+3. [Executing and deploying](#execution)
+4. [Reviewing and analizing](#review)
+
+
+## Project Definition and Scope <a name="scope"></a> 
+Every project must start with a basic introduction to the team and a kick off meeting with the client for the official presentations. Once this is done, the documentation must be started with the follwing information in a dedicated folder in Google Drive:
+
+### Basic Information <a name="basic"></a>
+    * Start and estimated end dates.
+    * General description.
+    * Initial documentation/mockups/wireframes... submited by the client.
```

Comment:
We should also add all the access keys required for the project (Meldium/other-service, amazon keys, mail services api keys, gmaps keys, etc.)